packages/teams-js/src/public/app.ts

```diff
@@ -687,26 +685,7 @@ export namespace app {
       return;
     }
 
-    if (GlobalVars.frameContext) {
```

Comment:
I am not sure, why we were un-initializing (setting null) only for some of the handlers. For example, all the handlers in remoteCamera, meetingRoom, videoEx etc are not reset.
In addition, some handlers are reset only during certain frameContexts. e.g. registerOnSaveHandler on L701 below.

Reply:
I'm willing to bet that these were just oversights and not part of some master plan.